charts/templates/namespace-global-sidecar.yaml

```diff
@@ -0,0 +1,310 @@
+            initialDelaySeconds: 1
+            periodSeconds: 2
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            {{- toYaml $gs.resources | nindent 12 }}
+          securityContext:
+            runAsUser: 1000
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: to-global-sidecar
+  namespace:  {{ $ns }}
+spec:
+  configPatches:
+    {{- range $f.wormholePort}}
+    - applyTo: VIRTUAL_HOST
+      match:
+        context: SIDECAR_OUTBOUND
+        routeConfiguration:
+          name: "{{ . }}"
+          vhost:
+            name: allow_any
+      patch:
+        operation: REMOVE
+    - applyTo: ROUTE_CONFIGURATION
+      match:
+        context: SIDECAR_OUTBOUND
+        routeConfiguration:
+          name: "{{ . }}"
+      patch:
+        operation: MERGE
+        value:
+          virtual_hosts:
+            {{- if $f.dispatches }}
+              {{- $svcPort := . }}
+              {{- range $f.dispatches }}
+              - domains: {{ toYaml .domains | nindent 12 }}
+                name: {{ .name }}
+                routes:
+                  - match:
+                      prefix: /
+                    route:
+                      {{- if eq .cluster "_GLOBAL_SIDECAR" }}
+                      cluster: outbound|{{$svcPort}}||global-sidecar.{{ $ns }}.svc.cluster.local
+                      {{- else }}
+                      cluster: {{ tpl .cluster $ }}
+                      # (dict "fence" $f "dispatch" . "root" $ "Template" (dict "BasePath" "xx"))
+                      {{- end }}
+              {{- end }}
+              {{ else }}
+              - domains:
+                  - '*'
+                name: to_global_sidecar
+                routes:
+                  - match:
+                      prefix: /
+                    route:
+                      cluster: outbound|{{.}}||global-sidecar.{{ $ns }}.svc.cluster.local
+              {{- end }}
+          request_headers_to_add:
+            - header:
+                key: "Slime-Orig-Dest"
+                value: "%DOWNSTREAM_LOCAL_ADDRESS%"
+              append: true
+            - header:
+                key: "Slime-Source-Ns"
+                value: {{ $ns }}
+              append: true
+    - applyTo: VIRTUAL_HOST
+      match:
+        proxy:
+          metadata:
+            LAZYLOAD_APP: GLOBAL_SIDECAR
+        context: SIDECAR_OUTBOUND
+        routeConfiguration:
+          name: "{{ . }}"
+          vhost:
+            name: to_global_sidecar
+      patch:
+        operation: REMOVE
+    - applyTo: ROUTE_CONFIGURATION
+      match:
+        proxy:
+          metadata:
+            LAZYLOAD_APP: GLOBAL_SIDECAR
+        context: SIDECAR_OUTBOUND
+        routeConfiguration:
+          name: "{{ . }}"
+      patch:
+        operation: MERGE
+        value:
+          virtual_hosts:
+            - domains:
+                - '*'
+              name: allow_any_alias
```

Comment:
when we organize changes in a single ef config, it seems that we need not no longer need to use this name. Adding back the original `allow_any` is ok.